pkg/engine/transform/transform.go

```diff
@@ -41,6 +41,7 @@ const (
 	vmExtensionType  = "Microsoft.Compute/virtualMachines/extensions"
 	nicResourceType  = "Microsoft.Network/networkInterfaces"
 	vnetResourceType = "Microsoft.Network/virtualNetworks"
+	asResourceType   = "Microsoft.Compute/availabilitySets"
```

Comment:
the rest of the code abbreviates availability sets as "vmas", let's make this `vmasResourceType` for consistency 

(not a blocking suggestion)

Reply:
(and asIndexes becomes vmasIndexes)

Reply:
updated! actually this change failed some unit tests. I'm looking...

Reply:
I think unit tests has been fixed with latest commit.